runner.py

```diff
@@ -1236,7 +1250,12 @@ def cleanup(self):
 
         print('Stopping metric providers')
         for metric_provider in self.__metric_providers:
-            metric_provider.stop_profiling()
+            # pylint: disable=broad-exception-caught
+            try:
+                metric_provider.stop_profiling()
+            except Exception as exc:
```

Comment:
Isn't a more specific catch possible?

Reply:
Same rationale here: Sensitive function path and we want to trigger in any case.